index.html

```diff
@@ -0,0 +1,368 @@
+
+<img src="images/horizontal-fold.svg" height="150px" alt="">
+<img src="images/vertical-fold.svg" height="150px" alt="">
+
+*<br>An illustration of two [=foldable-screen device=] concepts in two different [=postures=]. ([Source](https://w3c.github.io/screen-fold/))*
+
+<aside class="note">
+Traditional single-screen laptop form factor is excluded from the foldable device definition and web specifications that refer to this form factor specifically refer to it as a <dfn>traditional laptop</dfn>. Wikipedia has a more nuanced definition of laptop types, but for the purpose of [related web specifications](#related-web-specifications) [all laptop types](https://en.wikipedia.org/wiki/Laptop#Types) are considered traditional laptops.
+</aside>
+
+### Dual-screen device
+
+A <dfn data-export="">dual-screen device</dfn> has two or more physical internal screens made of solid materials placed together to form a larger foldable virtual screen. A defining characteristic of this form factor is a physical hinge that can rotate from 0 degree to at most 360 degrees that forms a physical gap in between the screens. The range of the hinge movement as well as the width of the gap varies depending on the hardware design. The screens separated by the physical hinge are made of solid materials, typically glass, and as such are not flexible. This design requires web developers to work around the fact there is a physical gap between the screens on both sides of the fold.
+
+![](images/dual-screen-device.png)
+*<br>A [=dual-screen device=] folded open. ([Source](TODO))*
+
+### Foldable-screen device
+
+A <dfn data-export="">foldable-screen device</dfn> has one or multiple screens made of foldable material that can bend so that the screen can fold onto itself. The hinge can typically rotate from 0 degree to 180 degrees depending on the flexible screen materials’ capability of sustaining the bend radius. The material around the area of the screen that can bend is often softer than on the other parts of the screen. To differentiate from a dual-screen device, a foldable-screen device does not have a physical gap between the screens that form the logical regions. This makes it easier for existing web content to be presented without visual distractions when the foldable-screen device is folded open into so-called flat posture.
+
+![](images/foldable-screen-device-laptop.png)
+*<br>A [=foldable-screen device=] as a laptop. ([Source](TODO))*
+
+![](images/foldable-screen-device-mobile.png)
+*<br>Another [=foldable-screen device=] as a mobile phone. ([Source](TODO))*
```

Comment:
what makes this device foldable-screen and not dual-screen?

Reply:
No gap between the screens.

Reply:
Not very visible from the picture.